pkgs/by-name/go/google-chrome/package.nix

```diff
@@ -243,6 +243,7 @@ let
         --prefix XDG_DATA_DIRS   : "$XDG_ICON_DIRS:$GSETTINGS_SCHEMAS_PATH:${addDriverRunpath.driverLink}/share" \
         --set CHROME_WRAPPER  "google-chrome-$dist" \
         --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--ozone-platform-hint=auto --enable-features=WaylandWindowDecorations}}" \
+        --add-flags "--simulate-outdated-no-au='Tue, 31 Dec 2099 23:59:59 GMT'" \
```

Comment:
Add a comment here that explains what it does. Like "Disables auto updates and browser outdated popup"

Reply:
I couldn't add a comment above the line or at the end; makeWrapper would barf.

Reply:
I think you can do something like:

```
        --add-flags ${lib.concatStringsSep " " [
          "${lib.escapeShellArg commandLineArgs}"
          # Disables auto updates and browser outdated popup
          "--simulate-outdated-no-au='Tue, 31 Dec 2099 23:59:59 GMT'"
        ]}
```

Reply:
I tried this but the final quoting seemed weird... can you help me with this or maybe we can forgo it for submission?

Reply:
I pushed the change and rebased to latest master. Please test this change. ([example](https://github.com/NixOS/nixpkgs/pull/319126#issuecomment-2329773287))

Reply:
But anyway, here's the trace of "hello world" passed as `commandLineArgs`.

![image](https://github.com/user-attachments/assets/418f336b-a7d5-45c4-8248-0e37f8230e69)